src/HassClient/Helpers/Json/JsonExtensions.cs

```diff
@@ -124,6 +133,8 @@ HassService getServiceFields(string service, JsonElement element)
                 var serviceFields = new List<HassServiceField>();

```

Comment:
Ah, so the current model was already deserialized manually for the most part

Reply:
yes